### DISCUSSION TITLE
Official search card

### DISCUSSION THREAD

**Discussion by Stooovie** [home-assistant/frontend#10205] [Dashboard]
I have entities that I don't always want in Lovelace but accessing them via Config - Entities is cumbersome and the "e" hotkey obviously doesn't work on mobile. 

At the moment I'm using [Pierre Stahl's Search card but it's rather buggy](https://github.com/postlund/search-card). 

I think basic functionality like this should be an official feature. Thanks.

**Comment by bramkragten** [comment]
So you would basically want the quick bar as a lovelace card? Or maybe being able to add a button that activates the quick bar would also work?

**Comment by Stooovie** [comment]
Yes! The aforementioned Search card is great as it actually shows states of the search results (this is cropped results from search for "bedroom" using the Search card):

![Screen Shot 2021-10-29 at 09 02 40](https://user-images.githubusercontent.com/10572373/139390492-cc575cee-1145-4bbb-a528-4adc311f4f8a.jpg)

But what you propose would be a good start.
